src/lib/useOnWindowFocused.ts

```diff
@@ -7,9 +8,11 @@ export const useOnWindowFocused = (
   deps: DependencyList,
 ): void => {
   useEffect(() => {
-    window.addEventListener("focus", cb);
-    return () => {
-      window.removeEventListener("focus", cb);
-    };
+    if (isBrowser) {
+      window.addEventListener("focus", cb);
+      return () => {
+        window.removeEventListener("focus", cb);
+      };
+    }
   }, deps);
```

Comment:
Könnte man sich nicht das ganze `useEffect` sparen?

```js
if (!isBrowser) {
  return;
}

useEffect(...);
```

Reply:
Tatsächlich habe ich zuerst darüber nachgedacht, aber aus folgenden Gründen verworfen:

1. Hooks darf man nicht bedingt ausführen. In diesem Fall ist das allerdings nicht so schlimm, da sich diese Bedingung (`isBrowser`) ja niemals zu Laufzeit ändert.

2. Ich habe den Hook in ein eigenes Module ausgelagert. Somit finde ich, ist es die Aufgabe des Hooks selbst, den ursprünglichen Fehler zu verhindern. Alternativ müsste der Hook entsprechend selber einen Fehler werfen, wenn `!isBrowser`.

Reply:
OK; klingt sinnvoll. 👌